src/virt.c

```diff
@@ -135,8 +164,8 @@ static enum plginst_field plugin_instance_format[PLGINST_MAX_FIELDS] =
 
 /* BlockDeviceFormat */
 enum bd_field {
-	target,
-	source
+        target,
+        source
```

Comment:
If you're fixing the indentation, might as well make it 2 spaces like the other enums :)